e2e/pages/events/createOrderEvent.page.js

```diff
@@ -1,17 +1,38 @@
 const {I} = inject();
 const judgeAndLegalAdvisor = require('../../fragments/judgeAndLegalAdvisor');
 const orders = require('../../fixtures/orders.js');
+const postcodeLookup = require('../../fragments/addressPostcodeLookup');
 
 module.exports = {
   fields: {
     title: '#order_title',
     details: '#order_details',
     orderTypeList: '#orderTypeAndDocument_type',
     directionsNeeded: {
-      yes: '#orderFurtherDirections_directionsNeeded-Yes',
-      no: '#orderFurtherDirections_directionsNeeded-No',
+      id: '#orderFurtherDirections_directionsNeeded',
+      options: {
+        yes: '#orderFurtherDirections_directionsNeeded-Yes',
+        no: '#orderFurtherDirections_directionsNeeded-No',
+      },
     },
     directions: '#orderFurtherDirections_directions',
+    epo: {
+      childrenDescription: {
+        radioGroup: '#epoChildren_descriptionNeeded',
```

Comment:
Yes and no?

Reply:
My test adds the description. Adding a reference to Yes / No is not necessary in my opinion since we are testing only the Yes scenario.